### PR TITLE
Add Aptos definition to landing page, begin to emphasize our ecosyste…

### PR DESCRIPTION
**Pull request by clay-aptos**
### Description

Define "Aptos" for everyone to understand
Emphasize our benefits
Link to Aptos Ecosystem list

### Test Plan

In-browser edit and review.

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/5637)

### CODE REVIEW

File: developer-docs-site/docs/index.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ hide_table_of_contents: true
 
 # Aptos Developer Documentation
 
-Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
+Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." And we at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family.
```
Comment (banool) [inline developer-docs-site/docs/index.md:11]: imo just: "We at", not "And we at"

File: developer-docs-site/docs/index.md [hunk 2]
```diff
@@ -8,7 +8,9 @@ hide_table_of_contents: true
...
 
-Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
+Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." And we at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 
+
+This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
```
Comment (banool) [inline developer-docs-site/docs/index.md:13]: Awesome

**Review by banool — APPROVED** [review]

**Review by rustielin — APPROVED** [review]

File: developer-docs-site/docs/index.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/index.md:13]: Thank you!

File: developer-docs-site/docs/index.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/index.md]: Done!

**Review by banool — APPROVED** [review]

**Review by movekevin — APPROVED** [review]

**Review by gregnazario — APPROVED** [review]

File: developer-docs-site/docs/index.md [hunk 3]
```diff
@@ -8,7 +8,9 @@ hide_table_of_contents: true
 
 # Aptos Developer Documentation
 
-Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.
+Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." We at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family.
```
Comment (davidiw) [inline developer-docs-site/docs/index.md:11]: This is technically owned by the Foundation. We just support it as the Labs. So we should make sure this is reflected as such. Let's make an issue here to follow up and ensure we respect the Foundation appropriately.

File: developer-docs-site/docs/index.md [hunk 4]
```diff
@@ -147,4 +149,4 @@ We are excited that you are here, and we look forward to getting to know you. We
 * [Forum](https://forum.aptoslabs.com/)
 * [Medium](https://medium.com/aptoslabs)
 * [Telegram](https://t.me/aptos_official)
-* [Twitter](https://twitter.com/AptosLabs)
\ No newline at end of file
+* [Twitter](https://twitter.com/AptosLabs)
```
Comment (davidiw) [inline developer-docs-site/docs/index.md:152]: Twitter should be updates to Foundation.

File: developer-docs-site/docs/index.md (hunk 3 above)
Comment (davidiw) [inline developer-docs-site/docs/index.md:11]:
```suggestion
Welcome! Aptos is a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." This site is here to help you grow a [web3 ecosystem project](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 
```

File: developer-docs-site/docs/index.md (hunk 4 above)
Comment (davidiw) [inline developer-docs-site/docs/index.md:152]:
```suggestion
* [Twitter](https://twitter.com/Aptos_Network)
```

**Review by davidiw — COMMENTED** [review]
gonna just land these updates since its been here for too long

File: developer-docs-site/docs/index.md (hunk 3 above)
Comment (clay-aptos) [reply developer-docs-site/docs/index.md]: > respect the Foundation

Issue filed and assigned to us:
https://github.com/aptos-labs/aptos-core/issues/5773

**Pull request merged**